database/migrations/2025_08_19_071608_create_project_change_requests.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use App\Enums\ChangeRequestStatus;
+use App\Enums\ChangeRequestType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('project_change_requests', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('project_id')->constrained('projects')->onDelete('restrict')->onUpdate('cascade');
+            $table->foreignId('requested_by')->constrained('users')->onDelete('restrict')->onUpdate('cascade');
+            $table->enum('request_type', ChangeRequestType::values());
+            $table->json('changes');
+            $table->enum('status', ChangeRequestStatus::values());
```

Comment:
remove status
add 
approvals (json)
request_status enum of RequestStatuses
created_by connected to users